vfs2provider-jdbctable/src/main/java/com/seeburger/vfs2/provider/jdbctable/JdbcTableRowFile.java

```
@@ -1008,7 +1011,7 @@ private void rollbackConnection(Blob blob, ResultSet rs, PreparedStatement ps, C
             }
             catch (Exception e)
             {
-                e.printStackTrace(System.err);
+                LOG.warn(e);
```

I would use („Ignoring exception while cleaning up connection“, e)“ or a similar message.

---

done